src/Files.App/Utils/Storage/Operations/FilesystemHelpers.cs

```diff
@@ -163,7 +161,6 @@ showDialog is DeleteConfirmationPolicies.PermanentOnly &&
 
 			// Execute removal tasks concurrently in background
 			var sourcePaths = source.Select(x => x.Path);
-			_ = Task.WhenAll(sourcePaths.Select(jumpListService.RemoveFolderAsync));
```

Comment:
Does this code need a replacement from our end?

Reply:
I do not think so. We do not want track jump list by ourselves, we just sync with Explorer and if you use the shell operation API, Explorer updates its jump list anyway.